src/loader/bgw_launcher.c

```diff
@@ -105,12 +93,21 @@ report_bgw_limit_exceeded(void)
 static void
 report_error_on_worker_register_failure(void)
 {
-	ereport(ERROR, (errcode(ERRCODE_INSUFFICIENT_RESOURCES),
-					errmsg("No available background worker slots."),
-					errhint("TimescaleDB background worker launcher shutting down. It will be restarted by the postmaster.")));
+	ereport(LOG, (errcode(ERRCODE_INSUFFICIENT_RESOURCES),
+				  errmsg("no available background worker slots."),
```

Comment:
`errmsg` should not end with period. See here: https://www.postgresql.org/docs/current/static/error-style-guide.html